py/py_cromwell_setup.py

```diff
@@ -75,13 +76,13 @@ def find_app_status(env):
     print(f'Checking status for CROMWELL app')
     app_name, app_status, proxy_url = check_for_app(env)
 
-    print(f'app_name={app_name}; app_status={app_status}')
+    configure_cromwell(env, proxy_url)
```

Comment:
previously we would not configure_cromwell if there was no app name.  why is it ok to do so now?

Reply:
so in case user had a cromwell server running the config was pointing to say serverA. that server is now deleted so rather than cromwell configuration pointing to old serverA i believe it should reflect the true state i,.e  empty string.